backend/api/tests.py

```diff
@@ -70,3 +70,13 @@ def test_post_create_view(self):
         self.assertContains(
             response, self.relative_profile_url, status_code=status.HTTP_201_CREATED
         )
+
+    def test_root_status_code(self):
```

Comment:
Since this test method doesn't have anything to do with `Post` logic, let's move it into its own class. You can name it `APIRootTestCase`.